src/arrays.js

```diff
@@ -1,61 +1,93 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index < array.length) {
+    return array[index];
+  }
+  if (index >= array.length) {
+    return array[index - array.length];
+  }
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  const newArray = [];
+
+  for (let i = 0; i < strings.length; i++) {
+    newArray.push(strings[i].toUpperCase());
+  }
+  return newArray;
 };
+// Could do with refactoring to make more concise ^^ (use a higher order function) //
 
 const reverseWordsInArray = strings => {
-  // your code here
+  const reverseWords = strings.map(word =>
+    word
+      .split('')
+      .reverse()
+      .join('')
+  );
+  return reverseWords;
 };
 
 const onlyEven = numbers => {
-  // your code here
+  let evenNumbers = [];
+  for(let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+      evenNumbers.push(numbers[i]);
+    }
+  }
+  return evenNumbers;
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+  let removeNth = array.slice(0, index).concat(array.slice(index + 1, array.length));
+  return removeNth;
 };
 
 const elementsStartingWithAVowel = strings => {
-  // your code here
+  let vowelStart = strings.filter(strings => /^[aeiou]/i.test(strings));
+
+  return vowelStart;
```

Comment:
Nice use of regex! This can be simplified a bit furhter to avoid creating a variable unnecessarily:
```suggestion
  return strings.filter(string => /^[aeiou]/i.test(string));
```